.github/config/rubocop_linter_action.yml

```diff
@@ -0,0 +1,67 @@
+# .github/config/rubocop_linter_action.yml
+
+# Description: The name of the check that will be created.
+# Valid Options: A reasonably sized string.
+# Default: 'RuboCop Action'
+check_name: "RuboCop Results"
+
+# Description: Versions required to run your RuboCop checks.
+# Valid options: RuboCop and any RuboCop extension, by default the latest gem version will be used. You can explicitly state that
+# (not required) or use a version number, like '1.5.1'.
+# Default:
+#   versions:
+#     - rubocop: 'latest'
+versions:
+  - rubocop
+  - rubocop-rails
+  - rubocop-minitest
```

Comment:
You don't need this one, you're using `RSpec` instead of `Minitest` 😉

Reply:
deleted now